index.html

```diff
@@ -243,6 +386,10 @@ <h5 class="h5-2">I'm always interested in hearing
                 <form action="https://formspree.io/f/xayzvjyw" method="post" class="form">
                     <input type="text" name="full_name" id="name" placeholder="Full name" required
                     maxlength="30">
+                    <input type="text" name="full_name" id="name-f" placeholder="First name" required
+                    maxlength="30">
+                    <input type="text" name="full_name" id="name-l" placeholder="Last name" required
+                    maxlength="30">
```

Comment:
- [ ] [OPTIONAL] I know we have been asking to follow the Figma as close as possible, here comes an exception. We are supposed to overlook the Figma in this case and use a single input for both the mobile and desktop versions. 🤝️

**Project requirements**
![Screenshot from 2023-05-06 07-36-47](https://user-images.githubusercontent.com/17243325/236600464-5ebf4b5e-76f8-4182-b910-da60e3c647ef.png)